.gitignore

```diff
@@ -4,6 +4,7 @@
 *.dll
 *.so
 *.dylib
+qed
```

Comment:
Why? this proyect doesn't have that folder, so it doesn't need to be ignored (As far as I know)

Reply:
It prevents accidentally upload to Github the QED binary automatically generated by the main.tf in order to provision the AWS instances with it.